roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -62,3 +62,26 @@
   args:
     chdir: "{{ drupal_core_path }}"
   when: drupal_member_block.changed is defined and drupal_member_block.changed
+
+- name: Set default solr server to point to CLAW core
+  command: drush -y config-set search_api.server.default_solr_server backend_config.connector_config.core CLAW
+  args:
+    chdir: "{{ drupal_core_path }}"
+  register: set_search_api_config
+  changed_when: "'Do you want to update' in set_search_api_config.stdout"
+
+- name: Get solr config files to copy
+  command: "find /var/www/html/drupal/web/modules/contrib/search_api_solr/solr-conf/6.x -type f"
```

Comment:
Can you make a variable in `roles/internal/webserver-app/defaults/main.yml` for the webserver root (ie. `/var/www/html`) and use that here. Just to make configuration easier if you have a different Apache root.
So this would become something like:
```
command: "find {{ webserver_document_root }}/drupal/web/modules/contrib/search_api_solr/solr-conf/6.x -type f"
```